### ISSUE TITLE
Search for spanish subtitles from opensutitles.com not working

### ISSUE THREAD

**Issue by AndDontBlink** [josdion/subbuzz#4]
Hi, your plugin is excelent but it's not working for opensubtitles.com for spanish subtitles :

````
[2023-04-09 13:21:26.898 -03:00] [INF] [119] subbuzz.Providers.SubBuzz: opensubtitles.com: Request subtitle for 'Shazam! Fury of the Gods', language=es-mx, year=2023, IMDB=tt10151854
[2023-04-09 13:21:27.059 -03:00] [INF] [109] subbuzz.Providers.SubBuzz: opensubtitles.com: Search duration: 0.189 sec. Subtitles found: 0
````
But for english it works

````
[2023-04-09 13:22:03.476 -03:00] [INF] [127] subbuzz.Providers.SubBuzz: opensubtitles.com: Request subtitle for 'Shazam! Fury of the Gods', language=en, year=2023, IMDB=tt10151854
[2023-04-09 13:22:04.220 -03:00] [INF] [82] subbuzz.Providers.SubBuzz: opensubtitles.com: Search duration: 0.773 sec. Subtitles found: 2
````
Also i can confirm that exists subtitles in spanish by searching manually on site or using another plugin for subtitles for opensubtitles.com .

I guess that its an issue with "es-mx" from jellyfin search because opensubtitles.com uses "es"

**Comment by josdion** [comment]
Hi @AndDontBlink! Jellyfin has two Spanish languages to select from: "Spanish; Latin" - "es-mx" and "Spanish; Castilian" - "es". Opensubtitles on the other hand support only one Spanish language with language code - "es". I'll fix this issue in the next version, but for now use "Spanish; Castilian" when you're searching for subtitles.

**Comment by AndDontBlink** [comment]
Thanks for your fast reply , i fail to mention before that also happens with Spanish;Castillian , here's a log with english, Castilian, Latin:

```
[2023-04-10 13:11:48.181 -03:00] [INF] [43] subbuzz.Providers.SubBuzz: subbuzz: Start subtitle search for ¡Shazam! La furia de los dioses.
[2023-04-10 13:11:48.194 -03:00] [INF] [43] subbuzz.Providers.SubBuzz: opensubtitles.com: Request subtitle for 'Shazam! Fury of the Gods', language=en, year=2023, IMDB=tt10151854
[2023-04-10 13:11:50.022 -03:00] [INF] [176] subbuzz.Providers.SubBuzz: opensubtitles.com: Search duration: 1.84 sec. Subtitles found: 2
[2023-04-10 13:11:50.026 -03:00] [INF] [43] subbuzz.Providers.SubBuzz: subbuzz: Search duration: 1.844 sec. Subtitles found: 2
[2023-04-10 13:12:10.661 -03:00] [INF] [208] subbuzz.Providers.SubBuzz: subbuzz: Start subtitle search for ¡Shazam! La furia de los dioses.
[2023-04-10 13:12:10.700 -03:00] [INF] [208] subbuzz.Providers.SubBuzz: opensubtitles.com: Request subtitle for 'Shazam! Fury of the Gods', language=es-mx, year=2023, IMDB=tt10151854
[2023-04-10 13:12:10.749 -03:00] [INF] [47] subbuzz.Providers.SubBuzz: opensubtitles.com: Search duration: 0.086 sec. Subtitles found: 0
[2023-04-10 13:12:10.750 -03:00] [INF] [208] subbuzz.Providers.SubBuzz: subbuzz: Search duration: 0.088 sec. Subtitles found: 0
[2023-04-10 13:12:22.042 -03:00] [INF] [47] subbuzz.Providers.SubBuzz: subbuzz: Start subtitle search for ¡Shazam! La furia de los dioses.
[2023-04-10 13:12:22.053 -03:00] [INF] [47] subbuzz.Providers.SubBuzz: opensubtitles.com: Request subtitle for 'Shazam! Fury of the Gods', language=es-mx, year=2023, IMDB=tt10151854
[2023-04-10 13:12:22.106 -03:00] [INF] [208] subbuzz.Providers.SubBuzz: opensubtitles.com: Search duration: 0.063 sec. Subtitles found: 0
[2023-04-10 13:12:22.107 -03:00] [INF] [47] subbuzz.Providers.SubBuzz: subbuzz: Search duration: 0.065 sec. Subtitles found: 0
```

Debuging Jellyfin seems that at least from web interface (android version has same problem) the search should be with "spa" in both cases (that's maybe why the two searchs are with "es-mx")

![subbuzz_select](https://user-images.githubusercontent.com/22651148/230945757-c59afcb1-b824-4fd3-a074-af3aa92ebd0b.png)
![subbuzz_english](https://user-images.githubusercontent.com/22651148/230946047-bb026f57-263b-488d-8532-277f12a23e19.png)
![subbuzz_castilian](https://user-images.githubusercontent.com/22651148/230946080-8ff363f3-606e-4ff2-be67-796d784542f1.png)
![subbuzz_latin](https://user-images.githubusercontent.com/22651148/230946103-1fb93ee6-9fa0-48b7-943d-59566cb64efd.png)

Maybe , there's another config on my Jellyfin that's changing "spa" into "es-mx" ? 

I change config in library downlaod language and metadata language to Castillian (are the only place that Latin/es-mx is respected) but still  subtitle downlaod uses es-mx instead of es (maybe old movies has language fixed and will work with new ones)

**Comment by AndDontBlink** [comment]
Tested lastest builds with fix and can confirm works perfertly, also i didnt notice but yify didnt work either but now i get subtitles from opensubtitles, subscene and yify (before only subscene)

**Comment by josdion** [comment]
Yes, I noticed there were an issue with yify too, so I fix it. The main problem is that Jellyfin always request subtitles for Spanish with two letter iso code "es-mx", no mater what you select from language list. Anyway the issue is fixed in master.

**Comment by josdion** [comment]
Search for Spanish subtitles should be fixed for podnapisi.net too.

**Comment by josdion** [comment]
Fixed in 1.0.5.0
